src/sbs/process/conversion.py

```diff
@@ -902,6 +903,25 @@ def convert_ECO_turbidity_val(
 
     return turbidity
 
+def convert_ECO_beta_val(
+    rawBeta: float,
+    ScaleFactor: float,
+    DarkVoltage: float,
+):
+    """Converts a raw value for backscatter channel on a ECO-FLNTU to β (1/m/sr) 
+
+    Args:
+        rawBeta(float): raw counts for digital, raw volts for analog
+        ScaleFactor (float): β/count for digital, β/V for analog
+        Vblank (float): dark counts for digital, V for analog
+
+    Returns:
+        float: converted beta in β (1/m/sr)
```

Comment:
The units are sort of confusing and make it look equivalent to `sr/m`. How about `1/(m*sr)` or `1*m^-1*sr^-1`. The latter's a bit cumbersome without better formatting though.

Sorry for the slight delay in getting to this btw

Reply:
I suggest `m-1 sr-1` or `1/m/sr`